dev-tools/scripts/releaseWizard.yaml

```
@@ -793,6 +799,7 @@ groups:
     vars:
       vote_close: '{{ vote_close_72h }}'
       vote_close_epoch: '{{ vote_close_72h_epoch }}'
+      release_candidate_location: '{{ dist_url_base }}/solr-{{ release_version }}-RC{{ rc_number }}-rev-{{ build_rc.git_rev | default("<git_rev>", True) }}'
```

To support shorter lines:
```suggestion
      release_candidate_rc_folder: 'solr-{{ release_version }}-RC{{ rc_number }}-rev-{{ build_rc.git_rev | default("<git_rev>", True) }}'
      release_candidate_location: '{{ dist_url_base }}/{{ release_candidate_rc_folder }}'
```